build/buildx/index.md

```diff
@@ -1,7 +1,9 @@
 ---
-title: Docker Buildx
+title: Buildx component
```

Comment:
Buildx is in other places referred to as the Docker CLI Buildx plugin, I think we should be consistent.

---------------------------------------
As for structure of the Build section (and page titles) here's a working proposal
----------------------------------------
It seems this will be a broad section, WDYT about (and since in Engine there is already a topic called Docker Buildx) structuring the content like:
Docker Build
1. (Build) Overview
2 About Buildx
 2.1. Installing Buildx
 2.2. About Buildx
 2.3. Using Buildx
 2.4. Using Bake